zmon-controller-ui/views/trialRun.html

```diff
@@ -159,7 +159,11 @@ <h1 class="no-margin">Trial Run</h1>
 
                     <div class="form-group">
                         <div class="col-sm-12 text-right">
-                            <button class="btn btn-primary" ng-click="TrialRunCtrl.save()"><i class="fa fa-fw fa-save" ></i>Save as New Check    </button>
+                            <button class="btn btn-primary" ng-click="TrialRunCtrl.save()">
+                                <i class="fa fa-fw fa-save" ></i>
+                                <span ng-if="alert.id">Save changes</span>
```

Comment:
Yes, I've been tempted to refactor this controller too, but for the moment chose to keep it like that.

To my understanding, TrialRun evaluates Alerts (a user must input an Alert Condition to evaluate), thus the controller handles an 'alert'.

With the addition of check save now a user can save a check, making it confusing.